platform/linuxbsd/os_linuxbsd.cpp

```diff
@@ -1180,6 +1180,10 @@ String OS_LinuxBSD::get_system_ca_certificates() {
 	return f->get_as_text();
 }
 
+String OS_LinuxBSD::get_default_android_sdk_path() const {
+	return OS::get_singleton()->get_environment("HOME") + "/Android/Sdk";
```

Comment:
Handle trailing slashes if present in the environment variable (very unlikely here, but could technically happen):

```suggestion
	return OS::get_singleton()->get_environment("HOME").path_join("/Android/Sdk");
```